.changeset/new-owls-perform.md

```diff
@@ -0,0 +1,5 @@
+---
+"react-magma": patch
```

Comment:
this needs to change

Reply:
Can you refresh? It is updated: 
![image](https://user-images.githubusercontent.com/91160746/161773425-84e325d0-953e-4bc9-b6f2-74f66ada61ac.png)

[d35e03f](https://github.com/cengage/react-magma/pull/615/commits/d35e03f99f550bbde561662e296686c54510cc5c)